ghwatch.py

```diff
@@ -305,28 +320,37 @@ def main():
     if args.exclude_reasons:
         exclude_reasons.update(args.exclude_reasons.split(','))
 
+    if auto_update:
+        pull_repositories(config)
+
     set_window_size()
     buttons = draw()
 
     Key.start(hide_cursor=True)
 
-    t = 0
+    refr_t = 0
+    meta_t = 0
     try:
         while True:
             # auto-refresh periodically
-            if t >= args.refresh_time:
+            if refr_t >= args.refresh_time:
                 buttons = draw()
-                t = 0
+                refr_t = 0
+
+            # auto-update (git pull) metadata-repo(s)
+            if auto_update and meta_t >= auto_update:
+                meta_t = 0
+                pull_repositories(config)
 
             # handle key input
             k = Key.get()
             if not k:
                 Key.input_wait(1.0)
-                t += 1
+                refr_t += 1
+                meta_t += 1
                 continue
             if k == 'escape':
                 break
-            # print(k, Key.mouse_pos)
```

Comment:
Was this comment inadvertently removed?

Reply:
It's not strictly part of this change but imo it's okay to remove it,
After all it is not a useful comment only a commented-out debug statement I accidentally left around :slightly_smiling_face: